MVCapplication/src/main/java/com/softserve/if072/mvcapp/controller/CartController.java

```diff
@@ -2,90 +2,65 @@
 
 import com.softserve.if072.common.model.Cart;
 import com.softserve.if072.common.model.dto.CartDTO;
-import com.softserve.if072.mvcapp.service.UserService;
+import com.softserve.if072.mvcapp.service.CartService;
 import org.apache.commons.collections.CollectionUtils;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.client.RestTemplate;
 
 import java.util.List;
 
 /**
- * The CartController class is used to retrieve appropriate resources from the
- * REST server
+ * The CartController class handles requests for "/cart" and renders appropriate view
  *
  * @author Igor Kryviuk
  */
 @Controller
 @RequestMapping("/cart")
-public class CartController {
-    private static final Logger LOGGER = LogManager.getLogger();
-    @Value("${application.restCartURL}")
-    private String restCartURL;
-    @Value("${application.restCartBouthtURL}")
-    private String restCartBoughtURL;
-    @Value("${application.restCartDeleteURL}")
-    private String restCartDeleteURL;
-    @Value("${cart.found}")
-    private String cartFound;
-    @Value("${cart.buyingSubmint}")
-    private String cartBoughtSubmint;
-    @Value("${cart.SuccessfullyOperation}")
-    private String successfullyOperation;
-
-    private RestTemplate template;
-    private UserService userService;
-
+public class CartController{
     @Autowired
-    public CartController(RestTemplate template, UserService userService) {
-        this.template = template;
-        this.userService = userService;
-    }
+    CartService cartService;
```

Comment:
I'll change to autowire via constructor wherever necessary in my code in the next commit.